README.md

```diff
@@ -51,3 +96,10 @@ Supported arguments:
 
 - `--module`: Relative path or absolute local or remote URL to the module entry point to check. For example,
   https://deno.land/x/grammy/mod.ts.
+
+## Github workflow
+
+This repository also contains a Github workflow for checking for issues and reporting them in the website's Github repository issue section.
+
+TODO: Explain how to configure the workflow.
```

Comment:
Do you want to do this before merging?

Reply:
No. I actually forgot the flow of doing this. I'll do it in another commit after I verify the steps, and link it to the original repository.